CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to Roslyn
+
+Guidelines for contributing to the Roslyn repo.
+
+## Submitting Pull Requests
+
+- **DO** ensure submissions pass all Jenkins legs and are merge conflict free.
+- **DO** ensure submissions meet functional and performance expectations, including scenarios for which the team doesn’t yet have open source tests. This means you may be asked to fix and resubmit your pull request against a new open test case if it fails one of these tests.
```

Comment:
Not sure this one translates well to a "DO" form. We're telling them to ensure it passes things they don't see until we tell them it's broken. And with no link to what those expectations are, we're telling them to follow rules they don't know. Let's consider just deleting this line, I'm not sure what value it brings.